NOTICE

```diff
@@ -1,6 +1,6 @@
 
 Apache Kyuubi (Incubating)
-Copyright 2021 The Apache Software Foundation.
+Copyright 2021-2022 The Apache Software Foundation.
```

Comment:
using and onwards

Reply:
We can not do that.
https://www.mail-archive.com/general@incubator.apache.org/msg75717.html

Reply:
thanks for sharing